### PR TITLE
COL-756, swimlane with community assets; for now, filter by recent, comments and likes

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-756
UX: https://drive.google.com/file/d/0B9vFhXfb6IlZYVc1aENROVdrc3c/view

Filters will change per requirements.

### CODE REVIEW

File: public/app/dashboard/splash.html [hunk 1]
```diff
@@ -18,9 +18,11 @@ <h3 data-ng-bind="isMyProfile ? 'My Classmates' : 'Classmates'"></h3>
 
   <div class="featured-assets-container" data-ng-if="isMyProfile || featuredAssets.length">
     <h3>
-      <span class="featured-list-name" data-ng-bind="isMyProfile ? 'My Assets' : 'Assets'"></span>
+      <span class="featured-list-name" data-ng-if="!featuredAssets.length">My Assets</span>
+
       <div class="featured-list-filters" data-ng-if="featuredAssets.length">
-        :
+        <span class="featured-list-name" data-ng-bind="isMyProfile ? 'My Assets:' : 'Assets:'"></span>
+
```
Comment (johncrossman) [inline public/app/dashboard/splash.html]: The change above fixes wonky rendering of `:`.

File: public/app/dashboard/splash.html [hunk 2]
```diff
@@ -58,7 +60,47 @@ <h3 data-ng-bind="isMyProfile ? 'My Classmates' : 'Classmates'"></h3>
...
+  <div class="featured-assets-container" data-ng-if="communityAssets.length">
+    <h3>
+      <div class="featured-list-filters" data-ng-if="communityAssets.length">
+        <span class="featured-list-name">Everyone's Assets:</span>
+
+        <a data-ng-click="sortCommunityAssets('recent')" data-ng-if="sortCommunityBy !== 'recent'">Recent</a>
+        <span data-ng-if="sortCommunityBy === 'recent'">Recent</span>
+        |
+        <a data-ng-click="sortCommunityAssets('likes')" data-ng-if="sortCommunityBy !== 'comments'">Most Discussed</a>
```
Comment (paulkerschen) [inline public/app/dashboard/splash.html]: @johncrossman Should this be `sortCommunityAssets('comments')`?

Comment (johncrossman) [inline public/app/dashboard/splash.html]: Holy impacts, Batman! You're right.  Fixin'...

**Comment by johncrossman** [comment]
@paulkerschen This is ready (again)

**Pull request merged**